README.md

```diff
@@ -213,6 +213,81 @@ class MyDataflowType extends AbstractDataflowType
 
 This example uses `ContentStructureFactory` to check if the content exists and returns the adequate `ContentStrucure` to pass to the content writer.
 
+## Use the NotModifiedContentFilter
+
+When updating contents, you might want to ignore contents where the update would not result in any actual changes in fields values. In that case, you can add the `NotModifiedContentFilter` as one of your steps.
+
+```php
+// In your DataflowType
+public function __construct(NotModifiedContentFilter $notModifiedContentFilter)
+{
+    $this->notModifiedContentFilter = $notModifiedContentFilter;
+}
+
+//[...]
+protected function buildDataflow(DataflowBuilder $builder, array $options): void
+{
+    //[...]
+    $builder->addStep($this->notModifiedContentFilter);
+    //[...]
+}
+```
+
+This filter compares each field value in the `ContentUpdateStructure` received to the fields values in the existing content object. If all values are identical, this filter will return `false`, otherwise, the `ContentUpdateStructure` will be returned as is.
+
+Not all field types are supported by this filter. Il a field type is not supported, values will be assumed different. If your dataflow is dealing with content types containing unsupported field types, it is better to simply not use the `NotModifiedContentFilter` to prevent unnecessary overhead. 
+
+### Supported field types
+- ezstring
+- ezauthor
+- ezboolean
+- ezcountry
+- ezdate
+- ezdatetime
+- ezemail
+- ezfloat
+- ezisbn
+- ezobjectrelation
+- ezobjectrelationlist
+- ezkeyword
+- ezselection
+- eztext
+- eztime
+- eztags
+- novaseometas
+- ezurl
+- ezmatrix
+- ezgmaplocation
+- ezrichtext
+
+If you want to add support for a field type, simply create your own comparator.
```

Comment:
What are the types of unsupported eZ fields?

```suggestion
### Add custom field comparator

If you want to add support for a field type, simply create your own comparator.
```

Reply:
The list is exhaustive. If a field type isn't on the list, it is unsupported.

If you're asking about which native field types are not supported, it is basically the binary ones (ezbinaryfile, ezimage, ezmedia) and ezuser.

Reply:
As new field types may be added in eZPlatform in the future, I'd rather list which field types are supported instead of which aren't.